.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,86 @@
+    pipelinesascode.tekton.dev/on-target-branch: "[*]"
+    pipelinesascode.tekton.dev/task-1: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cli-tests-in-container.yaml"
+    pipelinesascode.tekton.dev/task-2: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cli-tests.yaml"
+    pipelinesascode.tekton.dev/task-3: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/configure-operator.yaml"
+    pipelinesascode.tekton.dev/task-4: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/create-rp-launch.yaml"
+    pipelinesascode.tekton.dev/task-5: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/evaluate-test-suites.yaml"
+    pipelinesascode.tekton.dev/task-6: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/generate-build-artifacts.yaml"
+    pipelinesascode.tekton.dev/task-7: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/generate-ids.yaml"
+    pipelinesascode.tekton.dev/task-8: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-cluster-version.yaml"
+    pipelinesascode.tekton.dev/task-9: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-component-versions.yaml"
+    pipelinesascode.tekton.dev/task-10: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-cluster.yaml"
+    pipelinesascode.tekton.dev/task-11: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-serverless-operator.yaml"
+    pipelinesascode.tekton.dev/task-12: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-and-tag-imagestreams.yaml"
+    pipelinesascode.tekton.dev/task-13: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-operator-images.yaml"
+    pipelinesascode.tekton.dev/task-14: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/release-tests.yaml"
+    pipelinesascode.tekton.dev/task-15: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/run-tfa-classifire.yaml"
+    pipelinesascode.tekton.dev/task-16: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-results-to-dashboard.yaml"
+    pipelinesascode.tekton.dev/task-17: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-slack-notification.yaml"
+    pipelinesascode.tekton.dev/task-18: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/setup-testing-accounts.yaml"
+    pipelinesascode.tekton.dev/task-19: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/uninstall-cluster.yaml"
+    pipelinesascode.tekton.dev/task-20: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upload-artifacts.yaml"
+    pipelinesascode.tekton.dev/task-21: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-triggers-tests.yaml"
```

Comment:
We don't follow naming conventions in these tasks - task name and file name differ. Please rename all of them to `upstream-*` in plumbing repo.

Reply:
Sent MR for this https://gitlab.cee.redhat.com/tekton/plumbing/-/merge_requests/511

Reply:
merged, thanks